hack/ci-deploy.sh

```diff
@@ -43,23 +43,20 @@ kubectl apply -f"${deploy_dir}"/scylla-operator-member.clusterrole.yaml
 
```

Comment:
I can't comment on the previous lines but L28 should be gone and the cp above should use the new manifests